### ISSUE TITLE
Stock and product edit issue with multishop.

### ISSUE THREAD

**Issue by therampagerado** [thirtybees/thirtybees#1724]
Today I'm testing the 'simplified product edit function'.

I need this because I sell some shoes and I have 2 shops. When I change the quantity of those products I would need to change them in both of them. 

So far so good. I enter a product I make sizes combinations with the generator and exit. As of current edge the shop context is set to default shop (recent commit).

I go to quantities and I want to change the quantities and I'm greeted by this message:

![image](https://github.com/thirtybees/thirtybees/assets/11251370/e67f7e3a-3f5e-478e-b220-0fa65c28ec49)

Then I remember that I had stock sharing turned off because it says it resets all current stocks (which is a big no-no).

![image](https://github.com/thirtybees/thirtybees/assets/11251370/b64d3e4c-1d64-4372-95de-3486693a79db)

Can we somehow remove this stock reset as majority of use cases for multishop would be one shop, then add another, etc. And if their settings are not proper (as mine were not) this would reset a merchant's stock every single time and they have to start from scratch with their original shop stock levels in order to implement a second shop.

**Comment by getdatakick** [comment]
We could offer different ways to initialize quantities when enabling this option:

1) set all quantities to zero (current functionality)
2) use quantities from one selected shop 
3) use sum of quantities from all shops in the group

**Comment by therampagerado** [comment]
Yes, if there could be an options to choose from it would be nice.

**Comment by therampagerado** [comment]
Hello Petr, how deep is this fix in the pipeline? Currently my second shop is basically stuck because I have to update all products I created for it twice - once in the main shop, once in the second shop. Also with orders in my A shop the quantities gradually shift and they are not updated in my B shop, this is a problem as somebody might find the product they need in shop A and if it's out of stock they might order it from the B shop.

I imagine I'm not the only one.

Thank you!

**Comment by therampagerado** [comment]
Related to this issue is another one I found out today.

When I click on 'share stock' button all my stock level are not zeroed but are displayed as '--' in the product list:
![image](https://github.com/thirtybees/thirtybees/assets/11251370/3ee46f90-ee3a-428f-8eca-519f40a1fc72)
Quantity row in the db shows '0' and the same is shown also in the Quantities field inside the product.

When I go and manually add stock then revert to 0 it shows '0' in red circle as it should in the product list.

**Comment by therampagerado** [comment]
As an addition to the aforementioned issue with -- in the stock field I'm adding another. 

When activating 'Share available quantities to sell' all products switch to 'When out of stock: Deny orders'.

It would be nice to not touch this setting as some products can have custom value here.

**Comment by getdatakick** [comment]
Related to #1878
